### DISCUSSION TITLE
python

### DISCUSSION THREAD

**Discussion by salihukeelahh** [521xueweihan/HelloGitHub#2730] [推荐项目]
am very interested in learning python

**Comment by CatalinAsul13** [comment]
NICEEE good luck

**Reply by salihukeelahh** [reply]
are you any good with python

**Reply by ali-kanbar** [reply]
I am.
Need anything?

**Reply by salihukeelahh** [reply]
yes please, I could really use your help. btw am sorry for the late reply

**Reply by ali-kanbar** [reply]
What do you need?
A tutor?

**Reply by salihukeelahh** [reply]
yes, I need all mentor, tutor, a role model. and you seem to be an Experienced person  in programming and i would love it if you could be my role model and i as your disciple

**Reply by ali-kanbar** [reply]
Sure let's discuss it.
I prefer to discuss this somewhere else, I will paste my linkedin url profile, you could go there, take a look, and connect with me.
Of course I will accept the connection request and we could contact each other their.
this is the url: www.linkedin.com/in/ali-kanbar

Or you could just type my name : Ali Kanbar

**Reply by salihukeelahh** [reply]
alright bro i will do that just now, thanks for the support

**Reply by salihukeelahh** [reply]
hi bro i just made a connect on your profile. go and verify

**Reply by CatalinAsul13** [reply]
Good luck bro in learning,you will see is not so hard

**Reply by salihukeelahh** [reply]
thanks bro, hey i was thinking it won't be bad to have two mentors and two role models. 
would you also like to mentor me?

**Reply by CatalinAsul13** [reply]
I m  really busy in this moment,cause I have many exams at college after holidays season,maybe after if don t forget,but I think is late in February

**Reply by salihukeelahh** [reply]
alright bro just contact me when are done with your exams bro.
this is my whatsapp number 09032237648 just mesaage me when you are with your exams

**Reply by ali-kanbar** [reply]
Hello,
Bro you didn't send me a connection request,
maybe you went to the wrong profile?
Please make sure that you did.
Also I wanted to contect you on whatsapp, but what is your country code?

**Reply by salihukeelahh** [reply]
i did connect with you bro check again
and my country code is +234 Nigeria

**Reply by salihukeelahh** [reply]
my username on linkedin is salihu keelah

**Reply by salihukeelahh** [reply]
my whatsapp number +234 09032237648 Nigeria

**Reply by salihukeelahh** [reply]
hi bro @ali-kanbar i didn't hear from you again bro?

**Reply by ali-kanbar** [reply]
Hello Salihu,
I messaged you on Linkedin bro.

**Comment by Fillare** [comment]
I can help you learn python.

**Reply by salihukeelahh** [reply]
that's awesome. When should we start?

**Reply by Acmjack** [reply]
牛逼

**Reply by snail2sky** [reply]
> 牛逼

这人也太逗了。。。。好几年了 不知道她python 学的咋样了~

**Reply by Fillare** [reply]
> that's awesome. When should we start?

whenever you say

**Comment by longlon651** [comment]
Awesome, thanks for sharing.
